### PR TITLE
W800rf32

### PR DESCRIPTION
**Pull request by horga83**
**Description:**
Initial add of documentation for w800rf32 component and binary_sensor and switch platform.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#17897

## Checklist:

- [ ] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/binary_sensor.w800rf32.markdown [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+      a1:
+        name: motion_hall
+        off_delay: 5
+        device_class: Motion
+      a2:
+        name: motion_kitchen
+        device_class: Motion
+```
+
+Configuration variables:
```
Comment (klaasnicolaas) [inline source/_components/binary_sensor.w800rf32.markdown:40]: Please use the new format: [read more](https://developers.home-assistant.io/docs/en/documentation_create_page.html)

File: source/_components/w800rf32.markdown [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+To enable W800rf32 in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+w800rf32:
+  device: PATH_TO_DEVICE
+```
+
+Configuration variables:
```
Comment (klaasnicolaas) [inline source/_components/w800rf32.markdown:30]: Same

File: source/_components/binary_sensor.w800rf32.markdown [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+title: "W800rf32 Binary Sensor"
+description: "Instructions on how to integrate W800rf32 binary sensors into Home Assistant."
+date: 2018-10-16 12:45
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: w800rf32.png
+ha_category: Binary Sensor
+---
```
Comment (klaasnicolaas) [inline source/_components/binary_sensor.w800rf32.markdown:14]: missing `ha_release` and `ha_iot_class`

File: source/_components/switch.w800rf32.markdown [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+title: "W800rf32 Switch"
+description: "Instructions on how to integrate W800rf32 switches into Home Assistant."
+date: 2018-10-16 12:45
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: w800rf32.png
+ha_category: Switch
+---
```
Comment (klaasnicolaas) [inline source/_components/switch.w800rf32.markdown:14]: Same

File: source/_components/w800rf32.markdown [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+description: "Instructions on how to integrate a W800rf32(a) into Home Assistant."
+date: 2018-10-16 10:15
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: w800rf32.png
+ha_category: Hub
+ha_release: 0.79
+---
```
Comment (klaasnicolaas) [inline source/_components/w800rf32.markdown:14]: Same

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

**Comment by frenck** [comment]
Closing this PR, since the parent PR has been closed.

**Pull request closed without merging**